python/llm/scripts/env-check.sh

```diff
@@ -62,14 +62,21 @@ check_cpu_info()
 {
   echo "-----------------------------------------------------------------"
   echo "CPU Information: "
-  lscpu | head -n 17
-  
+  lscpu | grep -E "Model name|CPU(s)|MHz|Architecture|CPU\(s\):" | grep -v "NUMA node0 CPU(s):"
+}
+
+check_cpu_type()
+{
+  echo "-----------------------------------------------------------------"
+  echo "CPU type: "
+  sudo dmidecode -t 17 | grep DDR
```

Comment:
can we remove sudo?

Reply:
In this case, can we have other ways to do so?

Reply:
Currently no.